### PR TITLE
Adding free video links to supplement documentation

### PR DESCRIPTION
**Pull request by Gregg**
Added free video links which give readers a visual option to learn core Vue.js concepts.  These videos don't require a lot of context switching, they're quick and basic, and they apply to Vue 3.

I purposely did not add free video links for topics listed in #311 Vue School links, though we do have videos which cover these topics.

### CODE REVIEW

**Review by devdreamsolution — APPROVED** [review]

File: src/guide/component-basics.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Components Basics
 
+[Watch a free video about component basics on Vue Mastery](https://www.vuemastery.com/courses/intro-to-vue-3/components-and-props-vue3)
```
Comment (bencodezen) [inline src/guide/component-basics.md:3]:
```suggestion
```

File: src/guide/class-and-style.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Class and Style Bindings
 
+[Watch a free video about class & style bindings on Vue Mastery](https://www.vuemastery.com/courses/intro-to-vue-3/class-and-style-binding-vue3)
```
Comment (bencodezen) [inline src/guide/class-and-style.md:3]:
```suggestion
```

File: src/guide/forms.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Form Input Bindings
 
+[Watch a free video about forms on Vue Mastery](https://www.vuemastery.com/courses/intro-to-vue-3/forms-and-v-model-vue3)
```
Comment (bencodezen) [inline src/guide/forms.md:3]:
```suggestion
```

**Comment by bencodezen** [comment]
At this time the team is reviewing how videos should be best implemented on the docs site, closing this PR for now and will reach out when we're ready for another PR.

**Pull request closed without merging**